megamek/src/megamek/server/Server.java

```diff
@@ -19997,6 +19996,9 @@ private void resolveHeat() {
                 int rhsRoll = Compute.d6(2);
                 int targetNumber = 2;
                 switch (entity.getConsecutiveRHSUses()) {
+                    case 0:
+                        targetNumber = 2;
+                        break;
```

Comment:
This statement is kind of pointless, since the default value of targetNumber is 2.

Reply:
But after that, you can found that...

>                     case 6:
>                     default:
>                         targetNumber = 13; // Auto-fail

Is it better to modify the default value and put =13 to Case 6?

Reply:
Think I'm going to take a look at this after I get home, since I can't look at the full Server.java class via the web interface.

Reply:
I would use the constant TargetRoll.AUTOMATIC_FAIL instead of 13 for your default case.

Reply:
Actually, it isn't because of a default value statement in the switch. Default value is 13.

Reply:
Could someone change 13 to TargetRoll.AUTOMATIC_FAIL ? I can't fix it by the web browser and I am still not used to GitHub Desktop.